.github/workflows/main.yml

```diff
@@ -2618,9 +2618,9 @@ jobs:
 
       - name: Setup GraalVM {{ matrix.graal }}
         run: |
-          URL=https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${{ matrix.graal }}/graalvm-ce-java11-linux-amd64-${{ matrix.graal }}.tar.gz
+          URL=https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${{ matrix.graal }}/graalvm-ce-java17-linux-amd64-${{ matrix.graal }}.tar.gz
           curl -L $URL | tar -xz
-          GRAALVM=`pwd`/graalvm-ce-java11-${{ matrix.graal }}
+          GRAALVM=`pwd`/graalvm-ce-java17-${{ matrix.graal }}
           echo "JAVA_HOME=$GRAALVM" >> $GITHUB_ENV
```

Comment:
unfortunately this isn't going to work, please see:
https://github.com/apache/netbeans/issues/6925#issuecomment-2003319844

I think we should consider to start disabling tests until someone who is interested in polyglot features wants to look into this.

Reply:
https://github.com/apache/netbeans/pull/6369 essentially bumped the version as far as possible already